helm_deploy/manage-my-prison/values.yaml

```diff
@@ -14,6 +14,8 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: manage-my-prison-cert
     path: /
+    annotations:
+      external-dns.alpha.kubernetes.io/aws-weight: "0"
```

Comment:
~I'm probably reading this wrong but shouldn't the weight on this new Ingress in `live.` `100`, as in 100% of traffic going to this?~

Actually AWS Route53 documentation on `weight` says:
> If you set Weight to 0 for all of the records in the group, traffic is routed to all resources with equal probability.

So in our case it doesn't really matter as this would be the only resource 👌

Reply:
this is saying that 0% of traffic would go to a _different_ ingress. for apps that want to load-balance between clusters, there would be another annotation to specify the other ingress to send _this_ much traffic to

Reply:
oops i got it totally backwards. the weight is in fact for _this_ ingress.
https://github.com/ministryofjustice/manage-my-prison/pull/4